src/processing/script/index.ts

```diff
@@ -140,14 +140,14 @@ function applyChanges (script: string, changes: CodeChange[], isObject: boolean)
     if (!changes.length)
         return script;
 
-    changes.sort((a, b) => a.start - b.start);
+    changes.sort((a, b) => (a.start - b.start) || (a.end - b.end));
```

Comment:
What does it mean?

Reply:
This change needs for the following array:
```js
[
    { start: 0, end: 20, code: 'a=temp.a, b=temp.b' },
    { start: 0, end: 0, code: 'var temp=obj' }
] // changes
```